.rubocop_todo.yml

```diff
@@ -483,11 +452,6 @@ Style/RedundantBegin:
 Style/RedundantParentheses:
   Enabled: false
 
-# Offense count: 505
-# Cop supports --auto-correct.
-Style/RedundantSelf:
```

Comment:
I quite don't understand the reasoning here: what search are you using right now for that?